requirements/base-py36.txt

```diff
@@ -0,0 +1,248 @@
+# SHA1:510bd9c1a77e824ae6edd46c3362539b114f28bd
+#
+# This file is autogenerated by pip-compile-multi
+# To update, run:
+#
+#    pip-compile-multi
```

Comment:
Nice!
Do we have a story for how we keep these up to date? Or do humans do that for now?

Reply:
here is what was decided as far as i understand:
- use pip-compile-multi to pin things manually
- use dependabot ("new" pyup) to update things automatically

i'd raise this question with ben since he was the author of that rfc.

Reply:
i meant in this repo, not for our standards. I'm guessing dependabot is in the "future work" bucket?